rudder-core/src/main/scala/com/normation/rudder/reports/ReportingConfiguration.scala

```diff
@@ -107,5 +130,32 @@ class AgentRunIntervalServiceImpl (
       )
     }
   }
+
+  override def getNodeReportingConfigurations(nodeIds: Set[NodeId]): Box[Map[NodeId, ResolvedAgentRunInterval]] = {
+    for {
+      gInterval  <- readGlobalInterval()
+      gHeartbeat <- readGlobalHeartbeat()
+      nodeInfos  <- nodeInfoService.getAll()
+    } yield {
+      nodeIds.map { nodeId =>
+
+        if(nodeId == Constants.ROOT_POLICY_SERVER_ID) {
```

Comment:
Le 24 janvier 2015 15:04:22 GMT+01:00, Francois Armand <notifications@github.com> a écrit :
>> @@ -107,5 +130,32 @@ class AgentRunIntervalServiceImpl (
>>        )
>>      }
>>    }
>> +
>> +  override def getNodeReportingConfigurations(nodeIds: Set[NodeId]):
>Box[Map[NodeId, ResolvedAgentRunInterval]] = {
>> +    for {
>> +      gInterval  <- readGlobalInterval()
>> +      gHeartbeat <- readGlobalHeartbeat()
>> +      nodeInfos  <- nodeInfoService.getAll()
>> +    } yield {
>> +      nodeIds.map { nodeId =>
>> +
>> +        if(nodeId == Constants.ROOT_POLICY_SERVER_ID) {
>
>What is the correct thing to do ?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/Normation/rudder/pull/796/files#r23496507

To be consistent, it should set the value to 5 is isPolicyServer is true
We'll see later for the server roles
-- 
Envoyé de mon téléphone Android avec K-9 Mail. Excusez la brièveté.